pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -1885,8 +1886,8 @@ StudyPasswordRequirements:
       type: boolean
     maxIdenticalConsecutiveCharacters:
       type: integer
-      default: 2
-      minimum: 0
+      nullable: true
```

Comment:
@bskinner this property should be nullable.

Reply:
How can it be null?  If there's a default for the field, the value of the field is always defined.

Reply:
I think the `default` was a mistake, thus I'm correcting it.

Reply:
How are we expecting clients to handle a `null` value here, and how would the server be responding? Given the context, even if the server is returning `null`, I don't think it fits the API here- `null` does not make sense for an integer in this case, and should be regarded as a bug in the server's implementation as opposed to an issue with the API docs.

Reply:
A bit of reverse engineering here...I think `null` means "use as many identical consecutive chars as you like".

![Screen Shot 2020-02-27 at 4 02 48 PM](https://user-images.githubusercontent.com/1653048/75486558-bb364680-597a-11ea-9e08-ed903a729041.png)

Reply:
I think Zim's right -- if it's `null` then there's no max limit on consecutive chars. Maybe @raiks2 can provide more tips here.

Reply:
I withdraw my objection- you're both correct on this. I was hung up on it being a `minimum` value field, not a max. That said, I think we should still have the `minimum` attribute on the type- bounding this to all positive values >= 0.

Reply:
Okay, this one is a bit funny. I think Auth0 only allows it to be `2` so I think it's either `null` or `2`. I don't know why it's an `integer` -- it probably should have just been a `boolean`. Anways, added in `minimum`.